github/README.md

```diff
@@ -44,7 +44,7 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/githu
 * **STATE:** The state of the status. Can be one of the following `error`,
   `failure`, `pending`, or `success`.
 
-## Usage
+### Usage
```

Comment:
Not sure if I should be changing this, but it improves readability of the markdown for me.